eng/devices/android.cake

```diff
@@ -347,14 +347,13 @@ Task("uitest")
 	Information("Run UITests project {0}", PROJECT.FullPath);
 
 	AdbLogcat(new AdbLogcatOptions() { Clear = true });
-	AdbShell("'pm list packages -f | grep update'");
+	AdbShell("pm disable-user --user 0 com.google.android.configupdater");
```

Comment:
I'm curious what is this for? This is disabling the default user on a specific android app.

Reply:
We're seeing a consistent-ish set of crashes during our UI Test runs. After looking at the logcat files we're thinking this might be due to the simulator updating google play and killing the app, so we're trying a variety of things to see if it'll help.

I've attached the two logcat dumps here if you have any additional thoughts we can try. 


[logcat_uitest_2024-03-03-14-44-10.log.zip](https://github.com/dotnet/maui/files/14495978/logcat_uitest_2024-03-03-14-44-10.log.zip)

[InputTransparencyWhenRootIsTransparentMatrix-Android-UITestBaseTearDown-AppLogs-logcat-b1db6ff116464ab79af76ccc9b24adb9.log](https://github.com/dotnet/maui/files/14495979/InputTransparencyWhenRootIsTransparentMatrix-Android-UITestBaseTearDown-AppLogs-logcat-b1db6ff116464ab79af76ccc9b24adb9.log)

@jonathanpeppers is thinking that a module is getting updated and then killing the app

03-03 14:42:38.697  6932  6932 I ChimeraModuleLdr: Module config changed, forcing restart due to module com.google.android.gms.maps_dynamite
03-03 14:42:38.697  6932  6932 I Process : Sending signal. PID: 6932 SIG: 9

03-03 14:42:37.686  2725  3324 I Finsky  : [262] oai.b(41): IQ: Notifying installation update. package=com.google.android.gms, status=INSTALLED
03-03 14:42:37.686  2725  9491 I Finsky  : [385] hak.a(2): [ContentSync] finished, scheduled=true


So, far this PR has passed the Android UITests 4 times in a row after disabling this. Which might just be random "luck"

Reply:
Do you need google play on the device? We tend to use default images for Android Unit tests to avoid any extra "stuff" which play would add on. (Assuming they have a default image for the API level you are targeting)

Reply:
I think they have some tests/code related to GPS like maps.

Reply:
We'll eventually need maps and we do have tests related to GeoLocation
It'd be nice to just get this solved with google play vs putting it off until the future. 

if this fixes us for now then we're in a good place for later

Reply:
Hmmm it looks like it stopped running again with the same set of messages. 

I removed the code in the teardown that re-enabled it just to see if maybe that's not wise to do.
If it still doesn't work, then I'll change it to only install emulators without google play to see if that resolves. 
If it does, then that's at least a starting point to know what's causing issues